dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -1132,6 +1162,151 @@ public void shouldReturnParserContainerUUID() throws Exception {
         Assert.assertTrue(html.matches(regexExpected));
     }
 
+    /**
+     * Method to test: {@link com.dotmarketing.portlets.htmlpageasset.business.render.HTMLPageAssetRenderedAPIImpl#getPageHtml(PageContext, HttpServletRequest, HttpServletResponse)}
+     * Given Scenario: Create a page with File Container liked with relative path in the template
+     * ExpectedResult: should work
+     *
+     * @throws Exception
+     */
+    @Test
+    public void shouldRenderRelativeContainerPath() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)